site/guides/porch-ui-installation.md

```diff
@@ -16,8 +16,98 @@ To access the Configuration as Data UI with a docker container, you will need:
 *   [git](https://git-scm.com/)
 *   [docker](https://docs.docker.com/get-docker/)
 
+## Running on a GKE cluster
+
+This setup assumes that you have a GKE cluster up and running with porch installed, and that
+your current kube context is set to that GKE cluster. We would welcome contributions or feedback
+from people that have set this up in other clouds outside of GKE.
+
+First, find a published image in the [kpt-dev/kpt-backstage-plugins container registry](https://console.cloud.google.com/gcr/images/kpt-dev/global/kpt-backstage-plugins/backstage-plugin-cad?project=kpt-dev).
+For this example, we will use `gcr.io/kpt-dev/kpt-backstage-plugins/backstage-plugin-cad:v0.1.0`.
+
+Next, create a namespace called `backstage`:
+
+```sh
+kubectl create namespace backstage
+```
+
+Then, run the following command to set up the backstage deployment and service account.
+Change the image name and tag in the `newName` and `newTag` flags in the below `kpt fn eval` command to
+the ones you would like to use:
+
+```sh
```

Comment:
It would be nice if this used kpt to fetch the configuration, but the other components don't yet either, so I'm fine with this approach.

I filed an issue:
https://github.com/GoogleContainerTools/kpt/issues/3365